api/src/Entity/Camp.php

```diff
@@ -422,4 +423,53 @@ public function removeMaterialList(MaterialList $materialList): self {
 
         return $this;
     }
+
+    /**
+     * @param Camp      $prototype
+     * @param EntityMap $entityMap
+     */
+    public function copyFromPrototype($prototype, &$entityMap = null) {
+        parent::copyFromPrototype($prototype, $entityMap);
+
+        $this->campPrototypeId = $prototype->getId();
+        $this->name ??= $prototype->name;
```

Comment:
In der Lagerliste ist es dann komisch, wenn der gleiche Name 2 mal auftaucht.
Evtl ein Copy anhängen (...in welcher Sprache...?), wenn der Name vom prototype kommt?
Oder den Namen nicht kopieren?